src/Nethermind/Nethermind.Merge.Plugin/Handlers/V1/ForkchoiceUpdatedV1Handler.cs

```diff
@@ -107,21 +104,30 @@ public Task<ResultWrapper<ForkchoiceUpdatedV1Result>> Handle(ForkchoiceStateV1 f
                 {
                     _logger.Info($"Syncing... Unknown forkchoiceState head hash... Request: {requestStr}.");
                 }
-
                 return ForkchoiceUpdatedV1Result.Syncing;
             }
 
             BlockInfo blockInfo = _blockTree.GetInfo(newHeadBlock.Number, newHeadBlock.GetOrCalculateHash()).Info;
             if (!blockInfo.WasProcessed)
             {
+                BlockHeader? blockParent = _blockTree.FindHeader(newHeadBlock.ParentHash!);
+                if (blockParent == null)
+                {
+                    if (_logger.IsDebug)
```

Comment:
maybe Logger.IsInfo to be able to distinguish situations for now

Reply:
Ok